packages/components/src/components/post-header/post-header.tsx

```diff
@@ -121,6 +86,27 @@ export class PostHeader {
     if (!this.mobileMenuExtended) await this.mobileMenuAnimation.finished;
   }
 
+  get scrollParent(): HTMLElement {
```

Comment:
For another ticket: we could use this element to hide the overflow along the X axis as needed by the main navigation.

Reply:
True. Could you create that ticket please.

Reply:
I opened this ticket as urgent: #4793